references/cicd-sharedflow-pipeline/pom.xml

```diff
@@ -0,0 +1,224 @@
+			<releases>
+				<updatePolicy>never</updatePolicy>
+			</releases>
+		</pluginRepository>
+	</pluginRepositories>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<org.slf4j.simpleLogger.defaultLogLevel>info</org.slf4j.simpleLogger.defaultLogLevel>
+		<project.root.dir>${basedir}</project.root.dir>
+		<target.root.dir>${basedir}/target</target.root.dir>
+		<commit></commit>
+		<branch>a local</branch>
+		<apigee.apiversion>v1</apigee.apiversion>
+		<apigee.profile>${env}</apigee.profile>
+		<apigee.org>${org}</apigee.org>
+		<apigee.env>${env}</apigee.env>
+		<apigee.apitype>sharedflow</apigee.apitype>
+		<apigee.options>override</apigee.options>
+		<apigee.config.dir>${target.root.dir}/resources/edge</apigee.config.dir> <!-- For config plugin -->
+		<apigee.config.options>update</apigee.config.options> <!-- For config plugin -->
+	</properties>
+
+	<!-- This is where you add the environment specific properties under various profile names -->
+	<profiles>
+		<profile>
+			<!-- Apigee Edge -->
+			<id>apigeeapi</id>
+			<properties>
+				<apigee.deploy.plugin.version>1.2.3</apigee.deploy.plugin.version>
+				<apigee.config.plugin.version>1.4.7</apigee.config.plugin.version>
+				<apigee.hosturl>https://api.enterprise.apigee.com</apigee.hosturl>
+				<apigee.authtype>oauth</apigee.authtype> <!-- optional: oauth|basic(default) -->
+				<apigee.tokenurl>https://login.apigee.com/oauth/token</apigee.tokenurl> <!-- update to https://zoneName.login.apigee.com/oauth/token -->
+				<apigee.username>${username}</apigee.username> <!-- machine user -->
+				<apigee.password>${password}</apigee.password> <!-- machine password -->
+				<apigee.bearer>${token}</apigee.bearer>
+				<apigee.refresh>${refresh}</apigee.refresh>
+				<api.northbound.domain>${org}-${env}.apigee.net</api.northbound.domain>
+			</properties>
+		</profile>
+		<profile>
+			<!-- Apigee X and hybrid -->
+			<id>googleapi</id>
+			<properties>
+				<apigee.deploy.plugin.version>2.1.1</apigee.deploy.plugin.version>
+				<apigee.config.plugin.version>2.1.1</apigee.config.plugin.version>
+				<apigee.hosturl>https://apigee.googleapis.com</apigee.hosturl>
+				<apigee.bearer>${token}</apigee.bearer> <!-- this takes precedence over service account file -->
+				<apigee.serviceaccount.file>${file}</apigee.serviceaccount.file>
+				<api.northbound.domain>${api.northbound.domain}</api.northbound.domain>
+			</properties>
+		</profile>
+	</profiles>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>io.apigee.build-tools.enterprise4g</groupId>
+					<artifactId>apigee-edge-maven-plugin</artifactId>
+					<version>${apigee.deploy.plugin.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>com.apigee.edge.config</groupId>
+				    <artifactId>apigee-config-maven-plugin</artifactId>
+				    <version>${apigee.config.plugin.version}</version>
```

Comment:
we are inconsistent about versions - some versions are specified in properties and referred to in the plugin, and others like the `maven-resources-plugin` are set inline (e.g. `2.6`) - could we please be consistent?

Reply:
Thats how we have it in [cicd-pipeline pom](https://github.com/apigee/devrel/blob/main/references/cicd-pipeline/pom.xml#L99) as well